js/views/roomlistview.js

```diff
@@ -292,22 +292,23 @@
 		leaveGroup: function() {
 			//If user is in that room, it should leave that room first.
 			if (this.model.get('active')) {
-				OCA.SpreedMe.Rooms.leaveCurrentRoom();
-				OCA.SpreedMe.Rooms.showRoomDeletedMessage(true);
+				OCA.SpreedMe.Calls.leaveCurrentCall(true);
 				OC.Util.History.pushState({}, OC.generateUrl('/apps/spreed'));
+			} else {
+				// TODO(fancycode): When does this happen? The user is in a call
+				// where the call is not active?
```

Comment:
The user could be in a different call, then the call that is being left/deleted is not the active one. But I think this is theory only, because there is no UI to leave a call which is not active atm.